src/com/mellanox/cicd/Matrix.groovy

```diff
@@ -62,6 +62,45 @@ def forceCleanupWS() {
     run_shell(cmd, "Clean workspace")
 }
 
+Map getArchConf(config, arch) {
+    def k8sArchConfTable = getConfigVal(config, ['kubernetes', 'arch_table'], [:])
+
+    if (!k8sArchConfTable) {
+        config.logger.warn("getArchConf | kubernetes -> arch_table parameter is not defined, defaults will be used")
+        k8sArchConfTable['supported_arch_list'] = ['x86_64', 'aarch64']
```

Comment:
too many copy&paste.
lets define lines 70-72 as ```setupK8ArchDefaults()``` and call it always in line 66, then merge into it entries from config file.

Reply:
Fixed.